### PR TITLE
Minimal set on unit-tests for aim storage/SDK/server API

### PR DESCRIPTION
**Pull request by alberttorosyan**

### CODE REVIEW

File: aim/storage/query.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def match(
         metric_name: str = None
     ) -> bool:
 
-        context_proxy = AimObjectProxy(lambda: context.to_dict())
+        context_proxy = AimObjectProxy(lambda: context.to_dict() if context else {})
```
Comment (mihran113) [inline aim/storage/query.py:154]: do we need this though?  this won't throw an exception here, and in the `_check` method the exception will be caught and logged

Comment (mihran113) [inline aim/storage/query.py:154]: Do we need this though?  this won't throw an exception here, and in the `_check` method the exception will be caught and logged

Comment (alberttorosyan) [reply aim/storage/query.py:154]: Need to double-check; had a test failing because of this.

**Review by gorarakelyan — APPROVED** [review]
lgtm

File: aim/storage/query.py (hunk 1 above)
Comment (alberttorosyan) [reply aim/storage/query.py]: @mihran113 checked and fixed. The issue was with the logging in `except` block.

**Review by mihran113 — APPROVED** [review]
nothing to add, everything's great

**Pull request merged**